deny.toml

```diff
@@ -88,6 +88,7 @@ allow = [
   "BSL-1.0",
   "Unicode-DFS-2016",
   "OpenSSL",
+  "Unicode-3.0",
```

Comment:
Have you made sure it's compatible with Apache 2.0?

Reply:
I am not a lawyer 😭